.github/workflows/main.yml

```diff
@@ -98,14 +99,18 @@ jobs:
             ${{ env.TOOLS_ROOTFS }}
             ${{ env.TOOLS_ROOTFS }}.sha512
 
-  test-rollup-http-server:
-    runs-on: ubuntu-22.04
+  test:
+    runs-on: ubuntu-latest-8-cores
     steps:
       - uses: actions/checkout@v3
         with:
           submodules: recursive
 
-      - name: Test rollup-http-server and echo-dapp client
+      - name: Test libcmt mock on host
```

Comment:
`libcmt.a` was removed from the release process.
This makes it hard for other teams to use it.
We should add it back.